docker-compose.yml

```diff
@@ -9,8 +10,8 @@ services:
     container_name: member2
     image: hazelcast/hazelcast:3.12.5
 
-  hazelcast-quarkus1:
-    container_name: hazelcast-quarkus1
+  quarkus-service1:
```

Comment:
I went for `quarkus-service` since I wanted to underline that we're running Quarkus + `service`
@mesutcelik